ci/ubuntu.dockerfile

```diff
@@ -10,7 +10,7 @@ RUN echo "To invalidate cache"
 RUN apt-get update -y && apt-get install -y \
     python3-nacl \
     libindy-crypto=0.4.5 \
-    libindy=1.8.2 \
+    libindy=1.8.3~1070 \
```

Comment:
Seems we need to update `indy-crypto` to version `0.5.1` as well since this version of libindy includes it statically.